distribution/lib/Standard/Visualization/0.0.0-dev/src/Widgets.enso

```diff
@@ -27,9 +27,12 @@ get_widget_json value call_name argument_names uuids="{}" =
 
     read_annotation argument = Panic.catch Any handler=(log_panic argument) <|
         annotation = Warning.clear <| Meta.get_annotation value call_name argument
-        return_target err = err.payload.target
-        Panic.catch Not_Invokable handler=return_target
-            annotation value cache=cache
+        Panic.catch No_Such_Argument (annotation value cache=cache) caught_panic->
```

Comment:
This code change shouldn't be necessary if the API change was compatible.

Reply:
We wanted to change the code anyway - previously the `catch` was too broad - it was catching _any_ `Not_Invokable` error, not necessarily the correct one.

But yes I can remove the new type and add extra info to the current error. The code change will stay but will be less different.